pkg/kubelet/kuberuntime/labels.go

```diff
@@ -90,7 +93,20 @@ func newPodLabels(pod *v1.Pod) map[string]string {
 
 // newPodAnnotations creates pod annotations from v1.Pod.
 func newPodAnnotations(pod *v1.Pod) map[string]string {
-	return pod.Annotations
+	annotations := map[string]string{}
+
+	// Get annotations from v1.Pod
+	for k, v := range pod.Annotations {
+		annotations[k] = v
+	}
+
+	if kubecontainer.HasWindowsHostProcessContainer(pod) {
+		// While WindowsHostProcessContainers is in alpha pass 'io.microsoft.containers.hostProcess' annotation
```

Comment:
'io.microsoft.containers.hostProcess' is not the value in the constant above `microsoft.com/hostprocess-container`

Reply:
fixed